src/filter/facebl0r/CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
-set (SOURCES facebl0r.cpp)
-set (TARGET facebl0r)
+# facebl0r filter - requires OpenCV (and protobuf)
+# This filter can cause conflicts with applications that also use protobuf
+# such as MLT, so it can be disabled with -DWITHOUT_FACERECOGNITION=ON
```

Comment:
FYI MLT does not use protobuf directly; however, I think OpenCV may. I do not recall this old issue, but there have been various conflicts in the past because both MLT and OpenCV can have deep dependencies.